src/openzeppelin/token/erc1155/presets/ERC1155MintableBurnable.cairo

```diff
@@ -0,0 +1,250 @@
+# SPDX-License-Identifier: MIT
+# OpenZeppelin Contracts for Cairo v0.2.1 (token/erc1155/presets/ERC1155MintableBurnable.cairo)
+
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.uint256 import Uint256
+from starkware.starknet.common.syscalls import get_caller_address
+from starkware.cairo.common.math import assert_not_zero
+
+from openzeppelin.access.ownable.library import Ownable
+from openzeppelin.token.erc1155.library import ERC1155
+from openzeppelin.introspection.erc165.library import ERC165
+
+#
+# Constructor
+#
+
+@constructor
+func constructor{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }(uri: felt, owner: felt):
+    ERC1155.initializer(uri)
+    Ownable.initializer(owner)
+    return ()
+end
+
+#
+# Getters
+#
+
+@view
+func supportsInterface{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }(interfaceId: felt) -> (success: felt):
+    return ERC165.supports_interface(interfaceId)
+end
+
+@view
+func uri{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }(id: Uint256) -> (uri: felt):
```

Comment:
Token id is required here even if uri is not tokenId dependent, is it wanted? It doesn't seem to match the Solidity version ([source](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC1155/ERC1155.sol#L59)).

In addition, the tokenId is unused in the library.

Reply:
Regarding this point, after few tests, it seems to be required to ease interactions with marketplace clients.